package.json

```diff
@@ -15,19 +17,29 @@
   },
   "files": [
     "build",
-    "app.plugin.js"
+    "app.plugin.js",
+    "expo-module.config.json",
+    "ios"
   ],
   "keywords": [
     "react-native",
     "expo",
-    "hypertrack-sdk-react-native"
+    "hypertrack-sdk-react-native",
+    "HyperTrack",
+    "HyperTrackSDK"
   ],
   "repository": "https://github.com/hypertrack/sdk-expo",
   "author": "HyperTrack <help@hypertrack.com> (https://hypertrack.com)",
   "devDependencies": {
-    "expo-module-scripts": "^2.0.0"
+    "expo": "^46.0.13",
+    "@expo/config-plugins": "^4.1.5",
+    "expo-module-scripts": "^2.0.0",
+    "expo-modules-core": "^0.11.4"
   },
-  "dependencies": {
-    "@expo/config-plugins": "^4.1.5"
```

Comment:
@poterstar i moved this dependency to `devDepedencies` but lmk if that's not the right way to go about it

Reply:
I believe it should be in dependencies because without this plugin current plugin will not work, and DevDependencies are the packages that are needed during development.